FIRA:
  Camera: {Focal: 13.971313932573906, High: 650.0}
  Center: {Front: 270, Inner: 46, Outer: 255, X: 307, Y: 266}
  HSV:
    Bluerange: [253, 220, 233, 181, 201, 76]
    Greenrange: [202, 153, 255, 130, 255, 0]
    Redrange: [64, 0, 255, 119, 255, 0]
    Whiterange: [217, 151, 165, 0, 255, 56]
    Yellowrange: [140, 66, 255, 45, 255, 94]
  Scan:
    Parameter: [4, 3, 47, 85, 148, 200, 85, 91, 207, 213, 327, 333]
  blackItem: {angle: 3, gray: 50}
  whiteline: {angle: 4, gray: 100}
camera:
  image_raw:
    compressed: {format: jpeg, jpeg_quality: 80, png_level: 9}
    compressedDepth: {depth_max: 10.0, depth_quantization: 100.0, png_level: 9}
    theora: {keyframe_frequency: 64, optimize_for: 1, quality: 31, target_bitrate: 800000}
prosilica_driver: {auto_adjust_stream_bytes_per_second: true, auto_exposure: false,
  auto_gain: false, auto_whitebalance: false, binning_x: 1, binning_y: 1, exposure: 0.016666,
  exposure_auto_max: 0.5, exposure_auto_target: 50, fixed_rate: 15.0, frame_id: '',
  gain: 0, gain_auto_max: 19, gain_auto_target: 50, height: 480, stream_bytes_per_second: 45000000,
  trig_rate: 15.0, trig_timestamp_topic: '', trigger_mode: streaming, whitebalance_blue: 100,
  whitebalance_red: 100, width: 640, x_offset: 0, y_offset: 0}
rosdistro: 'indigo

  '
roslaunch:
  uris: {host_dei_iclab__41310: 'http://dei-iclab:41310/', host_dei_iclab__50475: 'http://dei-iclab:50475/'}
rosversion: '1.11.16

  '
run_id: 8b56e598-bb60-11e6-ab26-ecf4bb545821
